daemon/cluster/executor/container/container.go

```diff
@@ -139,6 +139,11 @@ func (c *containerConfig) labels() map[string]string {
 		labels[k] = v
 	}
 
+	// we apply service level labels, which could be obtained from ServiceAnnotations.Labels
+	for k, v := range c.task.ServiceAnnotations.Labels {
+		labels[k] = v
+	}
```

Comment:
Thanks @dnephin the pull request has been updated. Now the labels are added at the client side to the containers.